src/app/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import TextInput from '../../components/TextInput/TextInput';
+import SubmitButton from '../../components/SubmitButton/SubmitButton';
+import Typography from '../../components/Typography/Typography';
+import { Link } from 'react-router-dom';
+import styles from './LoginPage.module.css';
+
+export default function LoginPage(): JSX.Element {
+  return (
+    <main className={styles.login}>
+      <h2>
+        <Typography size="l">Einloggen</Typography>
+      </h2>
```

Comment:
👀 so many elements? `h2`, `h3` ;)

Reply:
I deleted them and now they are back, merge conflict party D: Time for the generalFixes5 branch.